src/http.c

```diff
@@ -651,7 +651,7 @@ int auth_log_in(struct tunnel *tunnel)
 	tunnel->cookie[0] = '\0';
 
 	if (username[0] == '\0' && tunnel->config->password[0] == '\0') {
-		snprintf(data, sizeof(data), "cert=&nup=1");
+		snprintf(data, sizeof(data), "");
```

Comment:
I guess this would have been sufficient instead:
```c
		data[0] = `\0`;
```
I'll change that in a later PR though.

Reply:
The later PR is #1235.